### PR TITLE
chore: clean up storybook stories and document storybook how-to

### PR DESCRIPTION
**Pull request by radium-v**
# Description

* Use storybook decorators to wrap all stories in a default `<fast-design-system-provider>`
* Remove existing provider elements from all stories
* Use side-effect imports to import components (removes the tree-shaking calls)
* Clean up some funky logic in some stories

## Motivation & context

Now that we're using an updated version of Storybook, we can use its features with more confidence. This reduces the complexity of writing a component's stories and will make it easier to start using other features offered by storybook.

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### CODE REVIEW

**Review by nicholasrice — APPROVED** [review]
Nice!

File: packages/web-components/fast-components/src/slider/fixtures/base.html [hunk 1]
```diff
@@ -1,342 +1,340 @@
...
+    <!-- Min above 0 -->
+    <h2>Min value greater than 0</h2>
+    <div style="width: 160px;">
+        <fast-slider min="5" max="15" step="1" value="10" style="min-width: unset;">
+            <fast-slider-label hide-mark="true" position="5">
+                5
             </fast-slider-label>
-            <fast-slider-label position="1">
-                1
+            <fast-slider-label hide-mark="true" position="15">
```
Comment (janechu) [inline packages/web-components/fast-components/src/slider/fixtures/base.html:86]: Why are these values changing?

**Review by janechu — COMMENTED** [review]
Looks good overall, only had one question in relation to changes to the stories for slider.

Edit: Also to note in future its easier to review smaller PRs as we are more likely to miss issues in large ones like this.

Edit 2: One strategy to consider if all of the changes should be performed at once is to create a feature branch and have the PRs be made against that.

File: packages/web-components/fast-components/src/slider/fixtures/base.html (hunk 1 above)
Comment (radium-v) [reply packages/web-components/fast-components/src/slider/fixtures/base.html:86]: 🤔 that's weird. Must be a formatter issue.

Comment (janechu) [reply packages/web-components/fast-components/src/slider/fixtures/base.html:86]: Yep, must be, double checked it against the original and it looks correct.

**Review by janechu — APPROVED** [review]

**Pull request merged**